src/AddressDriver.sol

```diff
@@ -28,7 +28,7 @@ contract AddressDriver is DriverTransferUtils, Managed {
     /// `driverId (32 bits) | zeros (64 bits) | addr (160 bits)`.
     /// @param addr The address
     /// @return accountId The account ID
-    function calcAccountId(address addr) public view returns (uint256 accountId) {
+    function calcAccountId(address addr) public view onlyProxy returns (uint256 accountId) {
```

Comment:
This change didn't _replace_ any `whenNotPaused`, like in the examples below?

Reply:
It didn't. Using `view` functions without the proxy can return confusing data, so if we want to use `onlyxProxy` at all, then it probably makes sense to secure all the functions, not only the state-modifying ones.